### DISCUSSION TITLE
关于在移动端使用AdGuard Home

### DISCUSSION THREAD

**Discussion by Dantezcx** [AdguardTeam/AdGuardHome#4472] [Q&A]
"从主屏幕中点击 ” 设置 “ 。
从左侧目录中选择 ” 无线局域网 “（移动数据网络环境下不支持修改 DNS ）。
点击当前已连接网络的名称。
在 DNS 字段中输入您的 AdGuard Home 服务器地址。"

我已经在软路由配置好了我的AdGuard Home 客户端
那么我在移动端的无线设置里面应该如何去手动添加DNS
例如：是直接填写使用192.168.0.1还是192.168.0.1:5058?
同理，如果外网有域名并且已经绑定ddns的话，是直接输入域名？还是需要加端口

**Comment by Dantezcx** [comment]
现在的进程就是，我有客户端已经配置完毕（软路由），有域名并且绑定了ddns，我想在外网环境（无线）在移动端使用我的AdGuard Home ，需要怎么设置呢

**Reply by cxw620** [reply]
不建议这么做，稳定性没保障，暴露公网也会有风险。
安卓原生应该不支持改DoT端口，低位端口频繁访问不知道运营商会不会封掉。实在想那么干，AdGuard客户端添加DoT或者DoH，输入DDNS的域名加DoT或者DoH的端口就行，例如`ddns.example:7777`
